GradeAnalytics/src/Main.java

```diff
@@ -231,8 +231,9 @@ public void actionPerformed(ActionEvent e) {
 			    chooser.setFileFilter(filter);
 			    int returnVal = chooser.showOpenDialog(null);
 			    if(returnVal == JFileChooser.APPROVE_OPTION) {
-			            result = chooser.getSelectedFile().getName();
+			            result = chooser.getSelectedFile().getAbsolutePath();
 			            textArea.setText(result);
+			            dataSet.createDataFromFile(result);
```

Comment:
We should be setting the result to the result from createDataFromFile and displaying that same thing for append